prow/config.yaml

```diff
@@ -10608,6 +10643,74 @@ periodics:
         defaultMode: 256
         secretName: ssh-key-secret
 
+- cron: "0 0,12 * * *" # mutual blocking with ci-kubernetes-e2e-gci-gce-scalability-stable1
+  agent: kubernetes
+  name: ci-kubernetes-e2e-gci-gce-scalability-release-1-7
+  spec:
+    containers:
+    - args:
+      - --timeout=140
+      - --bare
+      env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS
+        value: /etc/service-account/service-account.json
+      - name: USER
+        value: prow
+      - name: JENKINS_GCE_SSH_PRIVATE_KEY_FILE
+        value: /etc/ssh-key-secret/ssh-private
+      - name: JENKINS_GCE_SSH_PUBLIC_KEY_FILE
+        value: /etc/ssh-key-secret/ssh-public
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20171108-8b5aad9f-master
+      volumeMounts:
+      - mountPath: /etc/service-account
+        name: service
+        readOnly: true
+      - mountPath: /etc/ssh-key-secret
+        name: ssh
+        readOnly: true
+    volumes:
+    - name: service
+      secret:
+        secretName: service-account
+    - name: ssh
+      secret:
+        defaultMode: 256
+        secretName: ssh-key-secret
+
+- cron: "0 6,18 * * *" # mutual blocking with ci-kubernetes-e2e-gci-gce-scalability-release-1-7
```

Comment:
Same.

Reply:
I don't think they need to be blocking on each other. There seems to be enough quota in the project 'k8s-e2e-gci-gce-scale-1-4' and these are the only 2 jobs running there.

Reply:
But still please keep the cron strings disjoint just to prove that it works.

Reply:
updated, how about now?